3.6.2/rockcraft.yaml

```diff
@@ -0,0 +1,73 @@
+# Dockerfile: https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/blob/v3.6.2/images/Dockerfile
+name: sriov-network-device-plugin
+summary: SR-IOV Network Device Plugin rock
+description: |
+    A rock containing the SR-IOV Network Device Plugin for Kubernetes.
+
+    The SR-IOV Network Device Plugin is Kubernetes device plugin for
+    discovering and advertising networking resources in the form of
+    SR-IOV virtual functions (VFs), PCI physical functions (PFs),
+    Auxiliary network devices, in particular Subfunctions (SFs)
+license: Apache-2.0
+version: 3.6.2
+
+base: ubuntu@22.04
+build-base: ubuntu@22.04
+run-user: _daemon_
+
+platforms:
+  amd64:
+  arm64:
+
+environment:
+  APP_VERSION: 3.6.2
+
+# Services to be loaded by the Pebble entrypoint
+services:
+  install-cni:
+    override: replace
+    startup: enabled
+    command: bash /entrypoint.sh
+    on-success: shutdown
+    on-failure: shutdown
+
+parts:
+  add-golang:
```

Comment:
nit: I'm not sure if this is an official convention but I *think* that the parts should not be named like steps, e.g. I would just call this `go` or `build-deps`. 
https://github.com/canonical/k8s-snap/blob/main/snap/snapcraft.yaml#L25

Reply:
Done